perun-base/src/main/resources/perun-roles.yml

```diff
@@ -4464,6 +4500,8 @@ perun_policies:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   updateService_Service_policy:
```

Comment:
global MFA check for updating and creating

Reply:
still missing